WMF Framework/Wikipedia.xcdatamodeld/Wikipedia 3.xcdatamodel/contents

```diff
@@ -149,6 +149,8 @@
             <fetchIndexElement property="savedDate" type="Binary" order="ascending"/>
             <fetchIndexElement property="isExcludedFromFeed" type="Binary" order="ascending"/>
             <fetchIndexElement property="key" type="Binary" order="ascending"/>
+            <fetchIndexElement property="placesSortOrder" type="Binary" order="ascending"/>
+            <fetchIndexElement property="isDownloaded" type="Binary" order="ascending"/>
```

Comment:
@joewalsh ah! thanks for catching that - I wasn't entirely sure if this was the right index for housekeeper since it was missing placesSortOrder, but I didn't see anywhere else where the query matched as closely. This looks good 👍